README.md

```diff
@@ -1,2 +1,216 @@
-# To_Do_List
+# To Do List
+
```

Comment:
- [ ] The files in this directory should be in the root path. I cannot start the app from the root path. Kindly move all files in the `webpack-demo` folder to the root folder and remove the folder.

  ![image](https://github.com/AndreaM2429/To_Do_List/assets/36542986/de8039da-1abc-422c-a253-932a77f6b8b9)